server/sonar-server/src/main/java/org/sonar/server/startup/GeneratePluginIndex.java

```diff
@@ -26,16 +26,21 @@
 import java.io.Writer;
 import java.nio.charset.StandardCharsets;
 import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.CharUtils;
+import org.picocontainer.Startable;
```

Comment:
It's better to use `org.sonar.api.Startable`, which has exactly the same specification.